src/components/App/App.tsx

```diff
@@ -211,6 +215,13 @@ export function App(props: DocInnerProps): ReactElement {
     const {leftItems = [], rightItems = []} = header as NavigationData['header'];
     const headerWithControls = rightItems.some((item: {type: string}) => item.type === 'controls');
 
+    const controlSize: ControlSizes = ControlSizes.L;
+    const userSettings = {...settings, langs, onChangeLang};
+    const toc = data.toc;
```

Comment:
`controlSize` and `toc` a variables that is used once ? define them right where they were used